src/main/java/duke/entities/Event.java

```diff
@@ -0,0 +1,18 @@
+package duke.entities;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task{

```

Comment:
```suggestion
public class Event extends Task {
```
Missing a white space here